proposed/cache.md

```diff
@@ -348,6 +359,17 @@ interface CacheItemPoolInterface
     public function save(CacheItemInterface $item);
 
     /**
+     * Persists multiple cache items immediately.
+     *
+     * @param array|\Traversable $items
+     *   A traversable collection or array of CacheItemInterface[] items
```

Comment:
I would keep it as is but simply remove the square brackets after the interface name:

> A traversable collection or array of CacheItemInterface items